src/Nethermind/Nethermind.EvmPlayground/JsonRpcResponse.cs

```diff
@@ -38,11 +37,14 @@ public class JsonRpcResponse
     public class JsonRpcResponse<T>
     {
         [JsonProperty(PropertyName = "jsonrpc", Order = 1)]
-        public string Jsonrpc { get; set; }
+        public const string JsonRpc = "2.0";
+        
         [JsonProperty(PropertyName = "result", Order = 2)]
```

Comment:
I may not be understanding this code correctly, but should this have `NullValueHandling = NullValueHandling.Ignore` just like Error?  A JSON-RPC response can only have one of either `result` or `error` so it seems like they should be setup the same?

Reply:
Also, since the property is mutually exclusive, it feels like this should be broken up into two classes, one for `JsonRpcResponseSuccess` and one for `JsonRpcResponseError`

Reply:
I see another file that has both of these suggestions implemented.  I don't know why there are multiple files that define JsonRpc types, so perhaps the suggestion doesn't make sense for this one.

Reply:
EvmPlayground is a separate project taht we use during the EVM workshops. I started editing it by mistake. Since it is just a small tool that we use once every few months I did not want to spend time to refactor and make it use the standard RPC client classes. But the comments are valid and sorry that these changes  confused you.